### DISCUSSION TITLE
EurKEY

### DISCUSSION THREAD

**Discussion by TomazKarer** [manna-harbour/miryoku#53] [Q&A] [answered]
Hi!

I'm using [EurKEY](https://eurkey.steffen.bruentjen.eu/) layout. For my language (Slovene) we're using lots of Shift+AltGr combinations, for example 'Shift+AltGr 6, s' which produces 'š'. 

Is it possible to achieve this with Miryoku? I'm trying to achieve this but am failing to succeed.

**Comment by manna-harbour** [comment, marked as answer]
Hey!  AltGr is on both sides below Alt, so shift-altgr-6 would be hold right primary thumb, right index home, and right ring bottom row, in any order or simultaneously, and hit left index home.

If you have any unused keys on your keyboard you might like to map the dead key accents directly.  Or add combos to the base layer for the dead keys or a macro to produce the accented letter.

**Comment by TomazKarer** [comment]
Thank you, it works. I didn't spot the AltGr (it seems to be missing [here](https://github.com/manna-harbour/miryoku/blob/master/data/layers/miryoku-kle-reference.png) while included [here](https://github.com/manna-harbour/miryoku/blob/master/data/cover/miryoku-kle-cover.png).

**Comment by 10bn** [comment]
@TomazKarer 

Hi, I also use EuroKey and I can highly recommend using tabbed combos as Manna Harbour suggested. As a German, I've mapped keys such as "o+i" to "ö", "u+i" to "ü", "a+s" to "ä", and "s+d" to "ß". I've been using this mapping for quite a while now, and I'm quite happy with it.
